src/coreComponents/physicsSolvers/wavePropagation/sem/acoustic/secondOrderEqn/anisotropic/AcousticVTIWaveEquationSEMKernel.hpp

```diff
@@ -487,6 +489,7 @@ class ExplicitAcousticVTISEM : public finiteElement::KernelBase< SUBREGION_TYPE,
         stack.xLocal[ a ][ i ] = m_nodeCoords[ nodeIndex ][ i ];
       }
     }
+    stack.factor = 1.0;
```

Comment:
This is not used anymore, it can be removed

Reply:
Ho I see that it is used later on a derived class maybe a comment will be useful